pkg/config/runtime.go

```diff
@@ -151,8 +152,16 @@ func resolve(bag attribute.Bag, kindSet KindSet, rules map[rulesKey]*pb.ServiceC
 			glog.Warningf("%s attribute not found in %p", identityAttribute, bag)
 			return nil, fmt.Errorf("%s attribute not found", identityAttribute)
 		}
-	} else if scopes, err = GetScopes(attr.(string), identityAttributeDomain, scopes); err != nil {
-		return nil, err
+	} else {
+		attrStr, ok := attr.(string)
+		if !ok {
+			msg := fmt.Sprintf("%s attribute with value %v should be of type string", identityAttribute, attr)
+			glog.Warning(msg)
+			return nil, errors.New(msg)
```

Comment:
will this lead to double logging?  are resolve() errors logged as well, up the stack?

Reply:
Good catch. Removed the extra logging that I added. We already logging 2 times, this was making it even worse.

Starting gRPC server on port 9091
E0923 17:10:46.597749   38860 manager.go:201] unable to resolve config: destination.service attribute with value [10] should be of type string
E0923 17:10:46.598418   38860 grpcServer.go:307] Preprocess returned with: INTERNAL (unable to resolve config: destination.service attribute with value [10] should be of type string)